Lib/selectors.py

```diff
@@ -430,6 +430,9 @@ class PollSelector(_PollLikeSelector):
 
 if hasattr(select, 'epoll'):
 
+    NOT_EPOLLIN = ~select.EPOLLIN
```

Comment:
Perhaps make this private, since it is in the module namespace.
```suggestion
    _NOT_EPOLLIN = ~select.EPOLLIN
```

Reply:
sounds good